ProcessLib/HeatTransportBHE/BHE/FlowAndTemperatureControl.h

```diff
@@ -68,7 +68,15 @@ struct PowerCurveConstantFlow
     FlowAndTemperature operator()(double const T_out, double const time) const
     {
         double const power = power_curve.getValue(time);
-        return {flow_rate, power / flow_rate / heat_capacity / density + T_out};
+        if (power == 0)
+        {
+            return {0.0, T_out};
+        }
+        else
```

Comment:
No need for else branch after a return.

Reply:
Removed. Source code [here](https://github.com/HBShaoUFZ/ogs/blob/BHE_PNW/ProcessLib/HeatTransportBHE/BHE/FlowAndTemperatureControl.h).